app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "837971181421",
+    "firebase_url": "https://cs5520-firebase-assignment.firebaseio.com",
+    "project_id": "cs5520-firebase-assignment",
+    "storage_bucket": "cs5520-firebase-assignment.appspot.com"
+  },
```

Comment:
This file was downloaded from the firebase website. You don't need to download it but if you feel lost with what I've done here just go to [this link](https://firebase.google.com/docs/android/setup) and watch the video. I just followed that. I don't believe you need to make a firebase project. I've already done that. Not sure how to link it up to you but maybe it doesn't matter??